server/handlers.go

```diff
@@ -198,7 +198,7 @@ func openapi(w http.ResponseWriter, r *http.Request) {
 
 	var encodedContent []byte
 	if ct == JSONContentType {
-		encodedContent = wfs3.OpenAPI3SchemaJSON
+		encodedContent = wfs3.GenerateOpenAPIDocument()
 	} else {
```

Comment:
Why require a function call?  For tests this just makes for one more piece of setup to remember.

Reply:
The rationale here is that `init()` is set once and then the configuration passed/set/etc. in `main()`. Thus the OpenAPI description would not have a chance to be updated from config (service metadata, etc.).

Reply:
Make `wfs3.GenerateOpenAPIDocument()` only generate the document & return any errors and add `wfs3.GetOpenAPIDocument(encoding)`?